src/x86/sse.rs

```diff
@@ -868,6 +868,13 @@ pub unsafe fn _mm_prefetch(p: *const c_void, strategy: i8) {
     pref!(strategy)
 }
 
+/// Return vector of type __m128 with undefined elements.
+#[inline(always)]
+#[target_feature = "+sse"]
+pub unsafe fn _mm_undefined_ps() -> f32x4 {
+    mem::uninitialized()
```

Comment:
I think this may be causing a fault on i586?

Reply:
Thanks: when I saw #101, I should have fix `_mm_undefined_ps` but I did not.